index.html

```diff
@@ -1051,6 +1051,48 @@ <h2>Basic Concepts</h2>
 document.
       </p>
 
+      <section>
+        <h3>Getting Started</h3>
+
+        <p>
+This specification is designed to ease the prototyping of new types of
+<a>verifiable credentials</a>. Developers are provided with the following
+template that they can copy-paste into common <a>verifiable credential</a>
+tooling to start issuing, holding, and verifying prototype credentials.
+        </p>
+
+        <pre class="example nohighlight" title="A template for creating prototype verifiable credentials">
+{
+  "@context": ["https://www.w3.org/ns/credentials/v2"],
+  "type": ["VerifiableCredential", "MyPrototypeCredential"],
+  "credentialSubject": {
+    "mySubjectProperty": "mySubjectValue"
+  }
+}
+        </pre>
+
+        <p>
+It is expected that a developer would change `MyPrototypeCredential` above to
+the type of credential they would like to create. Since
+<a>verifiable credentials</a> talk about subjects, each property-value pair in
+the `credentialSubject` object expresses a particular attribute of the
+credential subject. Once a developer has added a number of these property-value
+combinations, the modified object can be sent to a <a>verifiable credential</a>
+issuer sofware and a <a>verifiable credential</a> will be created for the
+developer. From a prototyping standpoint, that is all a developer needs to do.
+        </p>
+
+        <p>
+Once a developer has prototyped their credential to a point where they believe
+all of the credential properties are stable, it is advised that they generate a
+vocabulary and context file for their application and publish it at a stable URL
```

Comment:
Is it possible to link to resources that teach developers how to generate the vocabulary and context file?

Reply:
Do you have a link to those resources? Unless they're exceedingly stable, we try to avoid linking to external content. For example, there is a video series I did that explains how to do this on YouTube as well as a few tutorials online to do so... however, those are on sites that may or may not stick around over the many years to decades that this specification might exist on the web (leading to broken links).

I guess we could link to the JSON-LD Specification? Would that work for you?

Reply:
Yes, linking to the json-ld spec would make sense to me.

I would love for specs to include explanatory videos in order to make them more accessible and easier to understand for folks, but that's outside of scope.